.github/workflows/sanitizer-build-and-test.yaml

```diff
@@ -0,0 +1,157 @@
+name: Sanitizer test
+on:
+  schedule:
+    # run daily 20:00 on master branch
+    - cron: '0 20 * * *'
+  push:
+    branches:
+      - prerelease_test
+jobs:
+  sanitizer:
+    name: PG${{ matrix.pg }} ${{ matrix.name }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["Sanitizer"]
+        pg: ["12.7", "13.3"]
+        os: ["ubuntu-20.04"]
+        build_type: ["Debug"]
+
+        # llvm-symbolizer crashes so we don't use clang until we resolve that
+        extra_packages: ["clang-10 llvm-10 llvm-10-dev llvm-10-tools"]
+        llvm_config: [llvm-config-10"]
+        clang: ["clang-10"]
+
+        # gcc-10 is the minimum version that can parse the suppression files barely adequately
+        cc: ["gcc-10"]
+        cxx: ["g++-10"]
```

Comment:
Why are these in the matrix? This seems like they should either be variables in the `env` section, or just used directly.
```suggestion
```